t/router/radixtree-sni.t

```diff
@@ -1254,3 +1254,81 @@ connected: 1
 failed to do SSL handshake: handshake failed
 --- error_log
 decrypt ssl key failed.
+
+
+
+=== TEST 28: set miss_head ssl certificate
+--- config
+location /t {
+    content_by_lua_block {
+        local core = require("apisix.core")
+        local t = require("lib.test_admin")
+
+        local ssl_cert = t.read_file("t/certs/miss_head.crt")
```

Comment:
The ssl creation will be failed if https://github.com/apache/apisix/issues/2816 is solved.

Reply:
Or add a TODO comment here.

Reply:
Add TODO comment and I will solve this case in `SSL certificate check` PR.